pineappl_capi/include/PineAPPL.hpp

```diff
@@ -174,19 +333,6 @@ struct Grid {
                                   key_val.raw);
   }
 
-  Grid() = delete;
-
-  Grid(const Grid &) = delete;
-
-  Grid(Grid &&) = delete;
-
-  Grid &operator=(const Grid &) = delete;
-
-  Grid &operator=(Grid &&) = delete;
```

Comment:
don't we need all those still? i.e. we want to deny copying any `Grid` object for users (and deny `GridBase` for developers)

Reply:
I guess this will be deprecated by https://github.com/NNPDF/pineappl/pull/328#discussion_r1944992515.